app/scripts/lib/transaction/metrics.ts

```diff
@@ -1087,6 +1089,7 @@ async function buildEventFragmentProperties({
     first_seen: time,
     gas_limit: gasLimit,
     transaction_contract_method: transactionContractMethod,
+    transaction_contract_address: transactionContractAddress,
```

Comment:
@dan437 – where do we ensure the "to" parameter corresponds to a contract and not an EOA?

Reply:
Great question. We only set up the transactionContractAddress if transactionType = TransactionType.contractInteraction;

Reply:
Where do we check this? I don't see the check in this PR – it looks like we are sending this property for all transactions, not just contract interactions.